test/e2e/scenarios/AddRemoveReplica/cmd_decrease_replica_multi_az.yaml

```diff
@@ -0,0 +1,54 @@
+id: "CMD_DECREASE_REPLICA_MULTI_AZ"
+description: "Decrease replica count for CMD with MultiAZ enable to disabled"
+resource:
+  apiVersion: $CRD_GROUP/$CRD_VERSION
+  kind: ReplicationGroup
+  metadata:
+    name: test$RANDOM_SUFFIX
+steps:
+  - id: "create_CMD_replication_group"
+    description: "Initial config"
+    create:
+      spec:
+        engine: redis
+        replicationGroupID: test$RANDOM_SUFFIX
+        replicationGroupDescription: Decrease replica count for CMD with MultiAZ enable to disabled
+        cacheNodeType: cache.t3.micro
+        numNodeGroups: 1
+        multiAZ: enabled
+        nodeGroupConfiguration:
+          - nodeGroupID: "0001"
+            replicaCount: 1
+            primaryAvailabilityZone: "us-west-2b"
+            replicaAvailabilityZones:
+              - "us-west-2c"
```

Comment:
Consider supplying these as replacement parameters. It will help in running the scenarios for other Region, AZ with minimal changes.